ledger-core/instrumentation/inslogger/test_helper.go

```diff
@@ -24,6 +24,8 @@ func NewTestLoggerExt(target logcommon.TestingLogger, suppressTestError bool, ad
 	if adapter != "" {
 		logCfg.Adapter = adapter
 	}
+	logCfg.Level = logcommon.DebugLevel.String()
+	logCfg.Formatter = logcommon.TextFormat.String()
```

Comment:
Why do you need text format?
JSON + build tag for text printer is much more readable

Reply:
build tag for json to text (with filed sort etc) is 
convlogtxt

Reply:
when you ran/debug tests on localmachine, json is not very easy to parse, and with convlog it's a hell of a mess, when it's json mixed with text